README.md

```diff
@@ -42,6 +42,12 @@
 
 ## Installation
 
+### Build Plugin
+
+If you'd like to use this package but not use any of the following steps, you can install [this build plugin](https://github.com/netlify/netlify-plugin-nextjs) that wraps this package! With the build plugin, you can skip all of the following installation and setup steps.
+
```

Comment:
hMmmMm. i think we should maybe hold off on this because the build plugin isn't supposed to be an alternative, it's meant to be the final iteration. at whatever point we start directing NoN users to the plugin, it should be a proactive push/suggestion to use the plugin instead of NoN and not a 'you can use this if you don't want to do extra steps'. it also at this moment has additional features over NoN (like custom netlify functions). but also it's not a big deal. would still be good to get it on people's radars. if you decide to merge the blurb as is, i'd put it under `### npm Package` for now.